deps.bzl

```diff
@@ -47,6 +51,34 @@ def gazelle_dependencies(
         remote = "https://github.com/bazelbuild/bazel-skylib",
     )
 
+    # Grab buildifier
+    _maybe(
+        http_file,
+        name = "buildozer_macos_arm",
+        urls = ["https://github.com/bazelbuild/buildtools/releases/download/5.0.1/buildozer-darwin-arm64"],
+        sha256 = "1d8a3b0e3a702630d36de012fd5cf93342af0fa1b5853643b318667345c84df0",
+        downloaded_file_path = "buildozer",
+        executable = True,
+    )
+
+    _maybe(
+        http_file,
+        name = "buildozer_macos_x86",
+        urls = ["https://github.com/bazelbuild/buildtools/releases/download/5.0.1/buildozer-darwin-amd64"],
+        sha256 = "17a093596f141ead6ff70ac217a063d7aebc86174faa8ab43620392c17b8ee61",
+        downloaded_file_path = "buildozer",
+        executable = True,
+    )
+
+    _maybe(
+        http_file,
+        name = "buildozer_linux_x86",
+        urls = ["https://github.com/bazelbuild/buildtools/releases/download/5.0.1/buildozer-linux-amd64"],
+        sha256 = "78204dac0ac6a94db499c57c5334b9c0c409d91de9779032c73ad42f2362e901",
+        downloaded_file_path = "buildozer",
+        executable = True,
+    )
+
```

Comment:
I think a better approach here, instead of using buildozer, is to enhance gazelle with a flag to add a set of strings as tag attribute under all the generated build target. 🤔

Reply:
I would agree, but there are code-paths through `go_repository` that will make use of pre-existing build files in the repositories. That is, we don't always generate the build files, but we always want to tag them. By using buildozer, we can do this in a consistent way.